Training/Setup/Patch/Data/AddCustomerAttributeCreditHold.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * Codifi_Training
+ *
+ * @copyright   Copyright (c) 2021 Codifi
+ * @author      Pavel Zelenevich <pzelenevich@codifi.me>
+ */
+
+namespace Codifi\Training\Setup\Patch\Data;
+
+use Magento\Framework\Setup\Patch\DataPatchInterface;
+use Magento\Framework\Setup\Patch\PatchRevertableInterface;
+use Magento\Framework\Setup\ModuleDataSetupInterface;
+use Magento\Customer\Setup\CustomerSetupFactory;
+use Magento\Customer\Setup\CustomerSetup;
+use Magento\Eav\Model\Entity\Attribute\SetFactory as AttributeSetFactory;
+
+class AddCustomerAttributeCreditHold implements DataPatchInterface, PatchRevertableInterface
+{
+    /**
+     * @var ModuleDataSetupInterface
+     */
+    private $moduleDataSetup;
+    /**
+     * @var CustomerSetup
+     */
+    private $customerSetupFactory;
+    /**
+     * @var AttributeSetFactory
+     */
+    protected $attributeSetFactory;
+
+    /**
+     * Constructor
+     *
+     * @param ModuleDataSetupInterface $moduleDataSetup
+     * @param CustomerSetupFactory $customerSetupFactory
+     */
+    public function __construct(
+        ModuleDataSetupInterface $moduleDataSetup,
+        CustomerSetupFactory $customerSetupFactory,
+        AttributeSetFactory $attributeSetFactory
+    ) {
+        $this->moduleDataSetup = $moduleDataSetup;
+        $this->customerSetupFactory = $customerSetupFactory;
+        $this->attributeSetFactory = $attributeSetFactory;
+    }
+
+    /**
+     * {@inheritdoc}
```

Comment:
In this case you should describe the functionality provided in this method because every data patch is created for specific purpose. For example, `Add 'Credit Hold' customer attribute` will be enough